src/server/terrain/structure_building_blocks.zig

```diff
@@ -135,6 +135,24 @@ pub const StructureBuildingBlock = struct {
 		}
 		return self;
 	}
+	pub fn initInline(sbbId: []const u8) !StructureBuildingBlock {
+		const blueprints = blueprintCache.get(sbbId) orelse {
+			std.log.err("['{s}'] Could not find blueprint '{s}'.", .{sbbId, sbbId});
+			return error.MissingBlueprint;
+		};
+		if(blueprints[0].childBlocks.len != 0) {
+			std.log.err("['{s}'] Inline structures cannot contain child blocks.", .{sbbId});
+			return error.InlineStructureCannotContainChildBlocks;
```

Comment:
Should this even be fatal? I think it might be useful to have these structures still spawn in the world in case the creator just wants to take a quick look how it fits in the world before actually defining the children.

Reply:
If you insist. If I allow that I need to generate a dummy SBB for every blueprint that doesn't have an SBB with an identical ID. This makes on-demand generation of inline SBBs actually important optimization.

Reply:
Couldn't you just not do the return error here and continue as you would normally?

Reply:
Also I would have to check if pasting code doesn't crash as a result, since it is blindly indexing into child Alias tables which there will be none of.

Reply:
Isn't that something that needs checking anyways? What does it do currently if a child is not bound and the file does exist?